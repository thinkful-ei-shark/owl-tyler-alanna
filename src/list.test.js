import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './list';
import store from './store';

it ('renders without crashing', ()=> {
    const div = document.createElement('div');

    ReactDOM.render(<List participants = {store.participants}/>, div);

    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<List participants = {store.participants}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });