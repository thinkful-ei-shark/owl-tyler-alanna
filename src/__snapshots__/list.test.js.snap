// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="list"
>
  <div
    className="participant"
  >
    <img
      alt=""
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <h3>
      Koren Templeton
    </h3>
    <p
      className="in-session"
    >
      On stage
    </p>
    ;
  </div>
  <div
    className="participant"
  >
    <img
      alt=""
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <h3>
      Ashla Attwool
    </h3>
    <p
      className="in-session"
    >
      On stage
    </p>
    ;
  </div>
  <div
    className="participant"
  >
    <img
      alt=""
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <h3>
      Caty Flucker
    </h3>
    <p
      className="in-session"
    />
    ;
  </div>
  <div
    className="participant"
  >
    <img
      alt=""
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <h3>
      Frank Runciman
    </h3>
    <p
      className="in-session"
    />
    ;
  </div>
  <div
    className="participant"
  >
    <img
      alt=""
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <h3>
      Axe Kubicka
    </h3>
    <p
      className="left-session"
    />
    ;
  </div>
</div>
`;
